src/pages/patterns/lead-space.mdx

```diff
@@ -83,15 +83,20 @@ On IBM.com pages, there are two layout options&mdash;left-aligned and center-ali
 
 ## Height standards
 
-The size of a lead space usually reflects the intent of the page and sets the right expectation for the user. Choose from these heights for the different purposes and use cases. To check out the available Lead space heights, see the Lead space component pages linked [below](#related-components-and-patterns).
+### Lead space left-aligned
+
+The size of a left-align lead space height usually reflects the intent of the page and sets the right expectation for the user. Choose from these heights for the different purposes and use cases. To check out more on Lead space heights, see the Lead space component pages linked [below](#related-components-and-patterns).
 
 | Height | Recommended usage |
 | :--- | :--- |
-| 560px and taller | Brand or immersive experience centric pages |
-| 480px, 400px | Branding and messaging centric pages |
-| 320px | Functional or navigational pages |
-| 240px | Minimal Lead space with simple title |
+| Super – 640px | Brand or immersive experience centric pages |
+| Tall – 560px | Marketing oriented pages |
+| Medium – 480px | Functional or navigational pages |
+| Short – 320px | Minimal lead space with simple title |
+
+### Lead space centered
 
+Lead space centered does not have fixed height variants and it is oriented more towards resolving functional and practical problems. If you want to transmit a more marketing-oriented message, please use the left-aligned lead space option.
```

Comment:
Some suggestions... we are lacking use cases at the moment to rigorously differentiate the usage of left-aligned vs centered, but here's my gut feeling: 

> Lead space centered does not have fixed heights. It is ideal for objective statements, such as academic research titles, and problem solving clues, such as `Find a product` or `Let's trouble shoot`. If you want to transmit a more marketing-oriented message, please use the left-aligned lead space option.

Reply:
Adjustment made.